descartes_light/test/utest.cpp

```diff
@@ -169,12 +169,49 @@ class SolverFixture : public ::testing::Test
   std::vector<std::size_t> zero_state_indices_;
 };
 
+template <typename SolverConfiguratorT>
+class ParameterizedSolverFixture : public ::testing::Test
+{
+public:
+  using FloatType = typename SolverConfiguratorT::FloatType;
+
+  ParameterizedSolverFixture() : state_cost(static_cast<FloatType>(1.0))
+  {
+  }
+
+  std::vector<typename WaypointSampler<FloatType>::ConstPtr> createSamplers(std::size_t dof,
```

Comment:
I would make this a stand-alone function and use it in both unit tests. This seems like something that might be useful for future unit tests as well

Reply:
Sounds good